motionplan/tpspace/ptgGroupFrame.go

```diff
@@ -143,6 +146,7 @@ func NewPTGFrameFromKinematicOptions(
 	pf.turnRadMillimeters = turnRadMillimeters
 	pf.trajCount = trajCount
 
+	// NOTE: Why are we creating these limits & why are they these values?
```

Comment:
Frames need to implement `DoF()`, which requires limits specifying ranges of valid values; these are they.

They correspond to how PTG inputs are used; {[which PTG to use by index, to 0 through solver count], [range of valid alpha values], [range of valid distance values]}

Reply:
got it, thanks! That makes a lot more sense